modules/express/src/errors.ts

```diff
@@ -40,3 +40,10 @@ export class IpcError extends Errors.BitGoJsError {
     Object.setPrototypeOf(this, IpcError.prototype);
   }
 }
+
+export class ExternalSignerConfigError extends Errors.BitGoJsError {
+  public constructor(message?: string) {
+    super(message || 'External signer configuration is invalid');
+    Object.setPrototypeOf(this, ExternalSignerConfigError.prototype);
```

Comment:
nit: this is no longer necessary and can be removed from all other errors defined here. Please remove in a follow up PR though, no need to do this clean up item here